codegen/method.go

```diff
@@ -1301,15 +1354,42 @@ func (ms *MethodSpec) setParseQueryParamStatements(
 		statements.append("\treturn")
 		statements.append("}")
 
-		if field.Required {
-			statements.appendf("requestBody%s = %s", longFieldName, identifierName)
+		target := identifierName
+
+		indent := ""
```

Comment:
nit: I think without proper indent `go fmt` will also take care of it

Reply:
`go fmt` formats the generated code, but not the golden files.